src/plugin/panel/popover-manager.c

```diff
@@ -143,6 +146,11 @@ void budgie_popover_manager_show_popover(BudgiePopoverManager* self, GtkWidget*
 	gtk_layer_set_keyboard_mode(popover_win, GTK_LAYER_SHELL_KEYBOARD_MODE_ON_DEMAND);
 }
 
+static gboolean on_focus_out(GtkWidget *widget, GdkEvent *event, GtkPopover *popover) {
+	gtk_widget_hide(GTK_WIDGET(popover));
+	return GDK_EVENT_PROPAGATE;
+}
```

Comment:
Thinking more on it, I think we should be extra safe and make sure that `popover` is actually valid before trying to use it.

```suggestion
static gboolean on_focus_out(GtkWidget *widget, GdkEvent *event, GtkPopover *popover) {
	g_return_val_if_fail(popover != NULL, GDK_EVENT_PROPAGATE);

	gtk_widget_hide(GTK_WIDGET(popover));
	return GDK_EVENT_PROPAGATE;
}
```

Reply:
agreed.  Tested and confirmed ok.